frontend/online-shop/src/App.tsx

```diff
@@ -0,0 +1,29 @@
+import { ThemeProvider } from "@mui/material";
+import { Container } from "@mui/system";
+import { Route, Routes } from "react-router-dom";
+import "./App.css";
+import Header from "./components/header/header.component";
+import ProductDetail from "./components/product-details/products-details.component";
+import { myTheme } from "./helpers/custom-theme";
+
+import HomePage from "./pages/home/home.page";
+import ProductsCollection from "./pages/products-collection/products-collection.page";
+import ShoppingCart from "./pages/shoping-cart/shopping-cart.page";
+
+const App = () => {
+  return (
+    <ThemeProvider theme={myTheme}>
+      <Header />
+      <Container>
+        <Routes>
+          <Route path="/" element={<HomePage />} />
```

Comment:
I would suggest to create a "routes.ts" file where you would expose a JSON object which contains all the routes. This way you can use the paths here and also reuse them whenever you want to redirect to one of those paths

Reply:
Here it would look like `path={routes.homepage}`